examples/Console/TestConsoleExporter.cs

```diff
@@ -39,7 +39,7 @@ private static object RunWithActivitySource()
             // and use Console exporter.
             using var tracerProvider = Sdk.CreateTracerProviderBuilder()
                     .AddSource("Samples.SampleClient", "Samples.SampleServer")
-                    .ConfigureResource(res => res.AddService("console-test"))
+                    .ConfigureResourceBuilder(res => res.AddService("console-test"))
```

Comment:
It seems the current code are bit random on the local variable naming.
Maybe use `resourceBuilder` or `resBldr` consistently across the repo?